src/common/common_init.cc

```diff
@@ -68,6 +68,8 @@ CephContext *common_preinit(const CephInitParameters &iparams,
       conf->set_val("keyring", "$mds_data/keyring", false);
     else if (conf->name.is_osd())
       conf->set_val("keyring", "$osd_data/keyring", false);
+    else if (conf->name.is_mgr())
```

Comment:
can we just hardwire the path of keyring of ceph-mgr to "$mgr_data/keyring" like mon, see https://github.com/ceph/ceph/blob/master/src/mon/Monitor.cc#L709 ? as we don't need to be backward compatible with previous releases here.